ERROR - 2021-12-14 23:11:40 --> Error connecting to the database: No connection could be made because the target machine actively refused it.

CRITICAL - 2021-12-14 23:11:40 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

#0 C:\xampp\htdocs\siakad\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\siakad\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\siakad\app\Models\Mdl_auth.php(13): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\siakad\app\Controllers\Auth.php(53): App\Models\Mdl_auth->login_user('admin', 'admin')
#4 C:\xampp\htdocs\siakad\vendor\codeigniter4\framework\system\CodeIgniter.php(824): App\Controllers\Auth->cek_login()
#5 C:\xampp\htdocs\siakad\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\xampp\htdocs\siakad\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\siakad\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\siakad\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
